src/boot.jsx

```diff
@@ -1,3 +1,4 @@
+import 'uswds'
```

Comment:
Needed for the [accordion](https://designsystem.digital.gov/components/accordions/).